### DISCUSSION TITLE
Does this run on a Raspberry Pi 4 with at least 2 gigs of ram?

### DISCUSSION THREAD

**Discussion by DeCodeerHeer** [amidaware/tacticalrmm#558] [Q&A] [answered]
I am managing a small non-profit organisation, but I recently moved out of my hometown where the organisation is vested. I believe they do wonderful work, and want to keep supporting them on IT basis. Hence why I am looking for a free RMM which can do the job. I would like to place a small server on-site which via I can manage the windows computers with on-site, and quickly resolve problems that occur during their operation hours. As a quick and cheap hosting solution, I was hoping I could use a Raspberry Pi 4 to host this software on. My question; is the software arm compatible, and are the specifications of the Pi 4 quick enough to run it?

**Comment by dinger1986** [comment, marked as answer]
Why bot run it on a free cloud platform? There's loads donated to charities, aws and azure do it for sure.

If not vultr costs only £5 a month.

No idea about the raspberry pi.

**Comment by wh1te909** [comment]
you could probably get it to work on a pi but you'll need to edit the install script for arm
as @dinger1986 said it's better to just get a cheap vps for a few bucks a month or use one of the free tier ones from the big cloud providers

**Comment by DeCodeerHeer** [comment]
Reverted to a suggested cheap VPS instead. Thanks for the help and suggestions! Software works very well.
